x-pack/legacy/plugins/ml/public/application/jobs/new_job/common/job_validator/job_validator.ts

```diff
@@ -60,6 +72,14 @@ export class JobValidator {
     scrollSize: { valid: true },
   };
   private _validating: boolean = false;
+  private _basicValidationResult$ = new ReplaySubject<JobValidationResult>(2);
+
+  private _jobConfigs$ = new Subject<JobCreator>();
```

Comment:
this name might be wrong. the subject is a JobCreator instance, not a jobConfig which is a JSON doc.

Reply:
yeah, will fix! forgot to rename after the latest changes